.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish
+on: [push]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3
+
+    - name: Build distribution
+      run: |
+        pip install -U setuptools wheel
+        python setup.py sdist bdist_wheel
+
+    - name: Publish to TestPyPI
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        password: ${{ secrets.TEST_PYPI_TOKEN }}
```

Comment:
This will take me a bit

Reply:
Should I move this to a new PR, if creating and configuring the TestPyPI user will take longer?

Reply:
Up to you. I hope to have it early next week, but I don’t know for sure.

Reply:
The secret is now defined :tada: